.gitignore

```diff
@@ -98,3 +98,5 @@ node_modules
 
 Tuist/Dependencies/SwiftPackageManager
 Tuist/Dependencies/graph.json
+
+.vscode
```

Comment:
What's the reason to add this one? The `.vscode` directory contains settings that belong to the repository and therefore I think we think we should revert this change.

Reply:
Ops, I have seen a new created file `.vscode/launch.json` (probably because of the swift plugin for VSCode) and I though it was not supposed to be committed

Reply:
Maybe we should add the `launch.json` as well (possibily in a separate PR)